common-lib/src/main/scala/com/gu/mediaservice/lib/BaseStore.scala

```diff
@@ -0,0 +1,54 @@
+package com.gu.mediaservice.lib
+
+import org.joda.time.DateTime
+
+import akka.actor.Scheduler
+import akka.agent.Agent
+
+import scala.concurrent.duration._
+
+import org.slf4j.LoggerFactory
+
+import com.amazonaws.auth.AWSCredentials
+import com.amazonaws.AmazonServiceException
+
+import org.apache.commons.io.IOUtils
+
+import _root_.play.api.libs.json._
```

Comment:
There is a `play` package in `common-lib` at the same level so it's necessary to be explicit here.